### PR TITLE
Filter the conversation list with the search term

### PR DESCRIPTION
**Pull request by nickvergessen**
Signed-off-by: Joas Schilling <coding@schilljs.com>

### CODE REVIEW

File: src/components/Navigation/ConversationsList/ConversationsList.vue [hunk 1]
```diff
@@ -38,12 +38,24 @@ export default {
...
 		},
+		filteredConversationsList() {
+			if (this.searchText === '') {
+				return this.conversationsList
+			}
+			return this.conversationsList.filter(conversation => conversation.displayName.indexOf(this.searchText) !== -1)
+		},
 		sortedConversationsList() {
-			return this.conversationsList.slice().sort(this.sortConversations)
+			return this.filteredConversationsList.slice().sort(this.sortConversations)
```
Comment (ma12-co) [inline src/components/Navigation/ConversationsList/ConversationsList.vue]: Why not do all this computation in the `conversationsList` computed prop? Or even in sorted but I mean.. Isn't 1 prop enough here?

Comment (nickvergessen) [reply src/components/Navigation/ConversationsList/ConversationsList.vue]: the reason I did it that way is that its only recalculating where necessary. But i guess most of the time the root data is changing anyway, so yeah let me merge them all in one

Comment (nickvergessen) [reply src/components/Navigation/ConversationsList/ConversationsList.vue]: Done

**Comment by nickvergessen** [comment]
Now case-insensitive

**Pull request merged**